ops/optimizer.rs

```diff
@@ -200,13 +200,33 @@ impl Transform {
         *ty = parse_quote! { *const #core::v8::fast_api::FastApiOneByteString };
         match str_ty {
           StringType::Ref => q!(Vars { var: &ident }, {
-            let var = unsafe { &*var }.as_str();
+            let var = match ::std::str::from_utf8(unsafe { &*var }.as_bytes()) {
+              Ok(v) => v,
+              Err(_) => {
+                unsafe { &mut *fast_api_callback_options }.fallback = true;
+                return Default::default();
```

Comment:
If memory serves, I had to do a separate `default` return value for pointer returning since there's no Default impl for a void pointer.